### PR TITLE
Preprocessing with some hardcoding (fixing this seperately)

### PR DESCRIPTION
**Pull request by Victoriam7**

### CODE REVIEW

File: backend/src/preprocessing.py [hunk 1]
```diff
@@ -2,147 +2,165 @@
...
+    bit_scale(packet)
+    dark_current_correction(packet, dimgs)
+    flat_field_w_gen(packet)
+    flat_fielding(packet)
+    registration(packet)
 
 Authors:
-    Brendan Grau <bsg8376@rit.edu>
+    Brendan Grau <https://github.com/Victoriam7>
+    Elija Cooper
```
Comment (egc2846) [inline backend/src/preprocessing.py:14]: there is an h in my name

**Review by egc2846 — DISMISSED** [review]

**Review by keenanm500 — DISMISSED** [review]
Nice, I like it, very clean. Just to confirm, this is all pre-processing besides working registration (and of course variable values)?

**Comment by egc2846** [comment]
> Nice, I like it, very clean. Just to confirm, this is all pre-processing besides working registration (and of course variable values)?

I believe so, Brendan can 100% confirm

File: backend/src/preprocessing.py (hunk 1 above)
Comment (Victoriam7) [reply backend/src/preprocessing.py:14]: Ill fix that. Sorry for the  mistake

Comment (Victoriam7) [reply backend/src/preprocessing.py]: Fixed

**Comment by Victoriam7** [comment]
This is complete preprocessing without working registration

**Review by egc2846 — APPROVED** [review]

**Review by sk4900 — APPROVED** [review]

**Review by jpakey99 — APPROVED** [review]

**Review by keenanm500 — APPROVED** [review]
You dismissed my review because you added an 'h' to Elijah's name lol

**Pull request merged**